src/CMakeLists.txt

```diff
@@ -103,6 +103,13 @@ if(BUILD_SHARED_LIBS)
     add_subdirectory(griddyn_shared)
 endif(BUILD_SHARED_LIBS)
 
+# ------------------------------------------------------------
```

Comment:
Are we anticipating that there will exist both versions of the C library for a little while at least?

Reply:
I don't know. Certainly, this one doesn't do everything the other one does, but I wasn't able to find anyone actually using the other one.